tools/mock-handling/src/recordings/common.ts

```diff
@@ -3,6 +3,7 @@ import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
 import fs = require("fs");
 import pick = require("lodash/pick");
 import isEmpty = require("lodash/isEmpty");
+import stringify from "json-stable-stringify";
```

Comment:
Detail: does this work? I mean this uses ESM import, while the other imports do not. If this works, we should consider using ESM import everywhere.

Reply:
yes it does (tested already on some recordings). imma going to change the imports everywhere